lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -342,6 +342,37 @@ actions to templates.
 A list of all built-in ones can be found in
 L<Mojolicious::Plugin::DefaultHelpers> and L<Mojolicious::Plugin::TagHelpers>.
 
+=head2 Plugins
+
+Plugins are application extensions that help with code sharing and
+organization. You can load a plugin with the keyword
+L<Mojolicious::Lite/"plugin">, which can omit the C<Mojolicious::Plugin::> part
+of the name, and optionally provide configuration for the plugin.
+
+  use Mojolicious::Lite;
+
+  plugin Config => {file => '/etc/myapp.conf', default => {foo => 'bar'}};
+
+  # Return configured foo value, or default if no configuration file
+  get '/foo' => sub {
+    my $c   = shift;
+    my $foo = $c->app->config('foo');
+    $c->render(json => {foo => $foo});
+  };
+
+  app->start;
+
+Some built-in plugins include L<Mojolicious::Plugin::DefaultHelpers>, which
```

Comment:
Mentioning an automatically loaded plugin here is a bit misleading. I guarantee it will be misunderstood.